### ISSUE TITLE
 aspnet core 3.1    linux        ThreadPool.SetMinThreads is not work

### ISSUE THREAD

**Issue by gaojiandonggg** [dotnet/aspnetcore#18410]
### To Reproduce

hello 
aspnet core 3.1  
 int processorCount = Environment.ProcessorCount;
 ThreadPool.SetMinThreads(processorCount * 5, processorCount * 100); 
run in docker  is not working by linux ,
but can working in  windows ,why?
 who can help me, thinks

**Comment by anurse** [comment]
I don't think this is specific to ASP.NET Core. I'm going to transfer this to dotnet/runtime and see if someone there can help.
